glTFLoaderUnitTests/SampleModelsTest.cs

```diff
@@ -203,5 +203,50 @@ public void WebGLSchemaLoad()
                 }
             }
         }
+
+        [Test]
+        public void BinaryLoad()
+        {
+            foreach (var file in Directory.EnumerateFiles(AbsolutePathToSchemaDir, "*.glb", SearchOption.AllDirectories))
+            {
+                try
+                {
+                    var len = new FileInfo(file).Length;
+
+                    Assert.IsTrue((len & 3) == 0);
+
+                    var jsonChunk = Interface.LoadModel(file);
+                    Assert.IsNotNull(jsonChunk);
+
+                    var binChunk = Interface.LoadBinaryBuffer(file);
+                    Assert.IsNotNull(binChunk);
+                    Assert.IsTrue((binChunk.Length & 3) == 0);
+
+                    var buffer = jsonChunk.Buffers[0];
+                    Assert.IsTrue(buffer.ByteLength <= binChunk.Length);
+                    // should we check padding as with jsonChunk? some reference files fail!

```

Comment:
See https://github.com/KhronosGroup/glTF/issues/1026. We should clarify spec on this issue.